test/attr/accessibility_proto.swift

```diff
@@ -1,91 +1,202 @@
-// RUN: %target-typecheck-verify-swift
-// RUN: %target-swift-frontend -typecheck -disable-access-control %s
+// RUN: %target-swift-frontend -typecheck -disable-access-control -package-name myPkg %s
```

Comment:
It looks like this used to test both the diagnostics and the disabling of access checking. I believe this dropped the check on the diagnostic. Can we keep a -verify check?

Reply:
Added it back in https://github.com/apple/swift/pull/64408